content/docs/iac/using-pulumi/build-a-component.md

```diff
@@ -2792,7 +2772,7 @@ We'll need to add the sources from the generated SDK output into the build sourc
         <dependency>
             <groupId>com.pulumi</groupId>
             <artifactId>pulumi</artifactId>
-            <version>[1.3,2.0)</version>
+            <version>1.16.1</version>
```

Comment:
Is this right? It looks like it is lower?

Reply:
@corymhall That would be a typo :)

Reply:
@corymhall That's what I said, too! Turns out [it's right](https://mvnrepository.com/artifact/com.pulumi/pulumi)